app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -135,6 +137,7 @@ class PatientSummaryScreen(
         .setActionTextColor(ContextCompat.getColor(context, R.color.green2))
         .setAnchorView(R.id.doneButtonFrame)
   }
+  private val detaches = detaches()
```

Comment:
Why was this extracted? It seems like it is not being used anywhere else.

Reply:
Oh, I forgot to add `takeUntil` for the `streamScreenResults` 😅

Reply:
This is not necessary since Mobius disposes the streams when the screen gets detached.

Reply:
We need to manually dispose streams only when they are not part of a Mobius loop.

Reply:
Ah okay 👍